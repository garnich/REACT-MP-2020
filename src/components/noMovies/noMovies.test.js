import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NoMovies from './noMovies';

Enzyme.configure({ adapter: new Adapter() });

describe('Test noMovies component', () => {
  it('<noMovies> test snapshot', () => {
    const wrapper = shallow(<NoMovies />)

    expect(wrapper).toMatchSnapshot()
  })
});