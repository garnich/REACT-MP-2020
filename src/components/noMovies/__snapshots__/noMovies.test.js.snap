// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test noMovies component <noMovies> test snapshot 1`] = `
<div
  className="no-movies"
>
  <p>
    No Movie Found
  </p>
</div>
`;
